changelogs/fragments/inventory_plugin.yaml

```diff
@@ -0,0 +1,3 @@
+---
+major_changes:
+  - Inventory plugin added. (https://github.com/ansible-collections/cloud.terraform/pull/22)
```

Comment:
Add the FQCN of the plugin please: `cloud.terraform.terraform_provider`

Reply:
Added